DEVREADME.md

```diff
@@ -0,0 +1,131 @@
+# Developers README
```

Comment:
Ik mis de splitsing tussen framework consumer developers and framework builder developers. e.g. Logs is voor consumers, tests runnen is voor builders

Reply:
En: liever niet `DEVREADME.md`.
Ook: willen we dit niet in de `docs` folder